packages/widget.vite-plugin.unstable/src/plugin.ts

```diff
@@ -669,6 +669,7 @@ function getPluginTransformHook(
     typeof plugin.transformIndexHtml === "object"
     && "transform" in plugin.transformIndexHtml
   ) {
+    // eslint-disable-next-line @typescript-eslint/no-deprecated
     return plugin.transformIndexHtml.transform;
```

Comment:
I think in general its good for it to be an error when we use deprecated types but I dont want to deep research vite plugins to understand if there is a consequence to this right now so we will save it as is.